### ISSUE TITLE
Update the PR template to allow for `Fixes #xxx`

### ISSUE THREAD

**Issue by hzoo** [babel/babel#4821]
https://github.com/babel/babel/pull/4813#issuecomment-258638438

I think we should change the PR template for `Fixed tickets #4516, #4686, #4324` so that it will actually affect the issues when the PR is closed since right now we forget to close out issues.

```
| Fixed tickets     | comma-separated list of tickets fixed by the PR, if any
```

**Comment by RichyHBM** [comment]
I'd be happy to do this, but I am unsure what gains there are. Is it just to change the wording?

**Comment by hzoo** [comment]
So it will close the issue and we don't forget about it https://help.github.com/articles/closing-issues-via-commit-messages/

**Comment by RichyHBM** [comment]
Ahh I see, I didnt realise GH had this feature! The docs however suggest `Fixes #4516, #4686, #4324` wouldnt work and instead would ned to be `Fixes #4516, Fixes #4686, Fixes #4324`

**Comment by hzoo** [comment]
Ah right - although I guess it has to be in the commit message itself so maybe instead it just needs to be on us as maintainers to added the missing (`fixes #1`) when merging actually..

**Comment by RichyHBM** [comment]
Ahh, this seems to suggest it can also be done via PR https://github.com/blog/1506-closing-issues-via-pull-requests
